clients/js/main.ts

```diff
@@ -28,4 +28,7 @@ if (isOutdated()) {
   );
 }
 
-yargs(hideBin(process.argv)).commandDir("cmds").strict().demandCommand().argv;
+yargs(hideBin(process.argv))
+  .commandDir("cmds", { recurse: true })
```

Comment:
wouldn't this pick up the subcommands automatically without needing the manual `Yargs` construction?

Reply:
how would it know what command those fns are linked to? in `cmds/sui/index.ts` we chain all of those fns and assign a command/description

it's also not possible to have a `exports.builder` in every file, because it ends up overwriting the last one for that subcommand sequentially